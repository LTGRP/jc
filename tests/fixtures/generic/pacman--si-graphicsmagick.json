[{"repository":"extra","name":"graphicsmagick","version":"1.3.43-1","description":"Image processing system","architecture":"x86_64","url":"http://www.graphicsmagick.org/","licenses":["MIT"],"groups":[],"provides":["Magick.so","libGraphicsMagickWand.so=2-64","libGraphicsMagick.so=3-64","libGraphicsMagick++-Q16.so"],"depends_on":["bzip2","freetype2","lcms2","libltdl","libpng","libsm","libtiff","libwebp","libxext","xz"],"optional_deps":[{"name":"jasper","description":"jp2 module"},{"name":"libwmf","description":"wmf module"},{"name":"libxml2","description":"msl, svg, url modules"},{"name":"ghostscript","description":"pdf, ps modules"},{"name":"libheif","description":"heic module"},{"name":"libjxl","description":"jpeg-xl module"}],"conflicts_with":[],"replaces":[],"download_size":"2.58 MiB","installed_size":"13.97 MiB","packager":"Caleb Maclennan <alerque@archlinux.org>","build_date":"Sat 23 Mar 2024 09:55:47 PM CET","validated_by":["MD5 Sum","SHA-256 Sum","Signature"],"download_size_bytes":2705326,"installed_size_bytes":14648606,"build_date_epoch":1711256147}]
