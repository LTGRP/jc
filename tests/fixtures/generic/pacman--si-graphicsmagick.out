Repository      : extra
Name            : graphicsmagick
Version         : 1.3.43-1
Description     : Image processing system
Architecture    : x86_64
URL             : http://www.graphicsmagick.org/
Licenses        : MIT
Groups          : None
Provides        : Magick.so  libGraphicsMagickWand.so=2-64  libGraphicsMagick.so=3-64  libGraphicsMagick++-Q16.so
Depends On      : bzip2  freetype2  lcms2  libltdl  libpng  libsm  libtiff  libwebp  libxext  xz
Optional Deps   : jasper: jp2 module
                  libwmf: wmf module
                  libxml2: msl, svg, url modules
                  ghostscript: pdf, ps modules
                  libheif: heic module
                  libjxl: jpeg-xl module
Conflicts With  : None
Replaces        : None
Download Size   : 2.58 MiB
Installed Size  : 13.97 MiB
Packager        : Caleb Maclennan <alerque@archlinux.org>
Build Date      : Sat 23 Mar 2024 09:55:47 PM CET
Validated By    : MD5 Sum  SHA-256 Sum  Signature
